core/src/main/java/com/google/errorprone/bugpatterns/javadoc/InvalidParam.java

```diff
@@ -101,6 +110,15 @@ public Description matchMethod(MethodTree methodTree, VisitorState state) {
     return Description.NO_MATCH;
   }
 
+  private static MethodTree getGeneratedConstructor(ClassTree classTree) {
```

Comment:
Given the current implementation:
```suggestion
  private static MethodTree getFirstConstructor(ClassTree classTree) {
```
This is not necessarily a generated constructor; see e.g. the `normalConstructor_record` test case.

That said, what we what, IIUC, is:
```suggestion
  private static MethodTree getCanonicalRecordConstructor(ClassTree classTree) {
```

But this does raise the question: can we reliable assume that the first constructor is always the canonical constructor? It might be better to explicitly test against `#isRecord`; will push a proposal.